crates/uv-publish/src/lib.rs

```diff
@@ -620,39 +730,11 @@ async fn handle_response(registry: &Url, response: Response) -> Result<bool, Pub
         ));
     }
 
-    // Detect existing file errors the way twine does.
```

Comment:
We scrap the index-specific twine code entirely and do the skip existing checks for any bad status